Modules/Scripted/ShapeAnalysisToolBox/DataImporter.py

```diff
@@ -487,6 +611,126 @@ def getTopologyAndConsistencyString(self, nodeName, inputSegmentName):
     """
     return self.getTopologyString(nodeName, inputSegmentName), self.getConsistencyString(nodeName, inputSegmentName)
 
+  #
+  #FreeSurfer tab functions
+  #
+  def initFreeSurferLUT(self,LUT_path):
```

Comment:
Function should probably be renamed as `loadFreeSurferLoopupTable` and implemented as a static function returning an object.  The idea is the also avoid side effect

Here is what you could do:
```
@staticmethod
def loadFreeSurferLoopupTable(filePath):
   loopupTable = dict()
   with open(filePath, 'r') as fp:
      for line in fp.readlines():
        (label_id, label_name, color_0, color_1, color_2) = line.split()
        loopupTable[label_id] = {'name': label_name, ...
   ...
  
   return loopupTable
```

Reply:
It may also be worth looking at this:
* [vtkMRMLFreeSurferProceduralColorNode](http://apidocs.slicer.org/master/classvtkMRMLFreeSurferProceduralColorNode.html)
* `slicer.util.loadColorTable`

Reply:
Why is it better to have a static function? what kind of side effect could appear?

Reply:
I was not aware that those tools exist, do you think I should use them instead of the current method?